File: .config/hiss/testdata/HISS-15/c/gap/positive-only-test.c
/* Only the positive dimension is asserted. The negative dimension (an inverted range) and
   the boundary dimension (INT_MIN, INT_MAX, value exactly at low and at high) are absent. */
#include <assert.h>

extern int clamp(int value, int low, int high);

int main(void) {
    assert(clamp(5, 0, 10) == 5);
    return 0;
}

File: .config/hiss/testdata/HISS-15/rust/gap/positive-only-test.rs
// Only the positive dimension is asserted. The negative dimension (an inverted range) and
// the boundary dimension (i32::MIN, i32::MAX, value exactly at low and at high) are absent.
#[cfg(test)]
mod tests {
    use super::clamp;

    #[test]
    fn inside_range() {
        assert_eq!(clamp(5, 0, 10), 5);
    }
}
